bentoml/_internal/bento/bento.py

```diff
@@ -33,10 +35,24 @@
 BENTO_PROJECT_DIR_NAME = "src"
 
 
+def _Bento_set_fs(bento: "Bento", _: t.Any, new_fs: "FS") -> "FS":
+    new_fs.makedir("models", recreate=True)
+    bento._model_store = ModelStore(new_fs.opendir("models"))  # type: ignore[reportPrivateUsage]
+    return new_fs
+
+
 @attr.define(repr=False)
 class Bento(StoreItem):
     _tag: Tag
-    _fs: FS
+    _fs: "FS" = attr.field(on_setattr=_Bento_set_fs)
+    _model_store: ModelStore
```

Comment:
```suggestion
    _model_store: Provider[BentoMLContainer.model_store]
```

Reply:
This is the model store inside the bento.

Reply:
btw this is not a correct usage.

Reply:
yep thats my bad